pandas/core/internals/construction.py

```diff
@@ -677,10 +657,22 @@ def sanitize_array(data, index, dtype=None, copy=False,
     return subarr
 
 
-def _try_cast(arr, take_fast_path, dtype, copy, raise_cast_failure):
-
+def _try_cast(arr, dtype, copy, raise_cast_failure):
```

Comment:
can you add types here?

Reply:
Nice removing the fast path arg!

Reply:
@jorisvandenbossche can you add types here?